### PR TITLE
[DFFS-27] Develop an E-commerce Store Database

### PR DESCRIPTION
**Pull request by Mayowa2020**

### CODE REVIEW

File: submissions/BukkyOyetimehin/database/init.sql [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+-- Customers Table
+CREATE TABLE customers (
+	id INT NOT NULL AUTO_INCREMENT,
+	first_name VARCHAR(20) NOT  NULL,
+	last_name VARCHAR(30) NOT NULL,
+	customer_email VARCHAR(20) NOT NULL,   
+	customer_password VARCHAR (20) NOT NULL,
+    created_at DATETIME,
+    PRIMARY KEY(id)
+);
```
Comment (Ireti203) [inline submissions/BukkyOyetimehin/database/init.sql:106]: You created two tables for customers?

**Review by Ireti203 — CHANGES_REQUESTED** [review]

File: submissions/BukkyOyetimehin/database/init.sql [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+('Danika Marshall', 'dmarshall@gmail.com', 'workflow%#1105', '226-813-4145', now()),
+('Linda McGrath', 'lmcgrath@gmail.com', 'shalom**203', '430-646-2808', now());
+
+SELECT * FROM admin_login;
+
+-- Products Table
+CREATE TABLE products (
+	id INT NOT NULL AUTO_INCREMENT,
+	product_name VARCHAR(100) NOT NULL,
+    Brand VARCHAR(100),
```
Comment (adaugochi) [inline submissions/BukkyOyetimehin/database/init.sql:48]: This column is not part of this table

File: submissions/BukkyOyetimehin/database/init.sql [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+	product_description VARCHAR(250) NOT NULL,
+	product_image BLOB NOT NULL,
+	product_price DECIMAL(10,2) NOT NULL,    
+    stock_level INT NOT NULL,
+    product_status VARCHAR(40) NOT NULL,
+    admin_login_id INT NOT NULL,
+    created_at DATETIME,
+    PRIMARY KEY(id),
+    FOREIGN KEY (category_id) REFERENCES categories (id),
+	FOREIGN KEY (admin_login_id) REFERENCES admin_login (id)
```
Comment (adaugochi) [inline submissions/BukkyOyetimehin/database/init.sql:59]: The table you are referencing here is different from what you created

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/BukkyOyetimehin/database/init.sql (hunk 2 above)
Comment (adaugochi) [reply submissions/BukkyOyetimehin/database/init.sql:53]: This column(brand) is not part of this table

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/BukkyOyetimehin/database/init.sql [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+);
+USE fashi_online_store;
+ALTER TABLE products CHANGE Brand brand VARCHAR(100);
```
Comment (adaugochi) [inline submissions/BukkyOyetimehin/database/init.sql:67]: `brand` should not be a column on the `products` table. Or, is there a place on the task where is it mentioned

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Review by Ireti203 — APPROVED** [review]

**Pull request merged**